lang/python/README.md

```diff
@@ -103,6 +103,26 @@ include $(TOPDIR)/feeds/openwrt-packages/lang/python/python3-package.mk
 
 Each maintainer[s] of external packages feeds is responsible for the local name, and relative inclusion path back to this feed (which is named `packages` by default).
 
+In case there is a need/requirement such that the local package feed is named something else than `packages`, one approach to make the package flexible to change is:
+
+```
+PYTHON_PACKAGE_MK:=$(wildcard $(TOPDIR)/feeds/*/python-package.mk)
```

Comment:
Does the `*` search subdirectories or would this need to be `PYTHON_PACKAGE_MK:=$(wildcard $(TOPDIR)/feeds/*/lang/python/python-package.mk)`?